source/localizable/components/the-component-lifecycle.md

```diff
@@ -67,7 +67,12 @@ you can use `didUpdateAttrs` to clear any error state that was built up from edi
 import Ember from 'ember';
 
 export default Ember.Component.extend({
-  errors: [],
+  errors: null,
```

Comment:
oh ok, just thought it would be good to list all variables like in a C# or Java class. will remove :)